style.css

```diff
@@ -0,0 +1,9 @@
+#navbar{
+  position:fixed;
+  top:0;
+}
```

Comment:
kindly add a one line space between your first and second CSS command so your code looks very neat.